sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/AttributesAssert.java

```diff
@@ -38,6 +38,11 @@
     return this;
   }
 
+  /** Asserts the attributes have the given key and value. */
+  public AttributesAssert containsEntry(AttributeKey<Long> key, int value) {
```

Comment:
why the `AttributeKey` parameter, rather than String `key` like the other methods?

Reply:
This is the equivalent of the above method which uses AttributeKey. The String version doesn't need an overload since there's no type parameter match, int is automatically converted to long.

Reply:
but why not just `(String, int)` like all the other specific methods?

Reply:
Because it isn't needed - you can call `(String, long)` with an int, but you can't call `(Foo<T>, T)` with an int

Reply:
And the reason that the `AttributeKey` form of the method is important is when using `SemanticAttributes.RESPONSE_LENGTH` for example